src/lib.rs

```diff
@@ -588,21 +676,35 @@ impl<T> StatefulWidget for Menu<T> {
             };
             let has_children = !item.children.is_empty();
 
-            let group_x_pos = x_pos;
+            let group_x_pos = {
+                if state.orientation.is_right() {
+                    x_pos
+                } else {
+                    let name_len = u16::try_from(item.name.len()).expect("name should be short");
```

Comment:
It might be difficult to actually work out what the problem is from this error message. Perhaps give more details. This is something a user will likely see, and the dev needs some info to repro.

Reply:
I'm not sure what I should do here, other than change "name" to "item name". This line just calculates the length of the name of root group, e.g. File, Edit, About in the example. I could show the problematic name in the .expect(...) but I'm not sure if this is a good idea since the length of it would be bigger than u16::MAX. I could truncate it though if that's what you want

Reply:
It seems reasonable to truncate this instead of erroring. I'm not sure I'd want to use a UI with menus that are 65536 chars long ;)